lib/fetching/bot-master.js

```diff
@@ -176,6 +189,9 @@ BotMaster.prototype._add = function(bot) {
   var self = this;
   this.bots.push(bot);
   this.emit('botMaster:addedBot');
+  this.on('settingsUpdated:' + bot.source.media, function() {
+    bot.contentService._reloadSettings();
```

Comment:
I have a suggestion: emit an event called just, `settingsUpdated` and pass a param indicating the media type. Then, catch that event in one place per process and reload the settings object once. Then let the bots also watch for that event and read from the updated settings object if appropriate.

Currently, if you have 10 Facebook bots and someone updates the facebook auth token, your code will (I think) reload the settings file 10 times...